osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1395,6 +1395,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/rac.svg"
       }
     },
+    {
+      "description": "Building Consensus among Hong Kong People",
```

Comment:
Does this describe the token or does it describe the chain? I don't see any enter in the chain registry to compare it to. Please register this to the Cosmos Chain Registry
https://docs.osmosis.zone/integrate/registration.html#how-to-register-onto-the-cosmos-chain-registry
https://github.com/cosmos/chain-registry

Reply:
This is a DAO token in juno
https://daodao.zone/dao/juno1vntnqevwk47escuxrfrdk99070vs0gczyyx0wa2pp08j25n0ee6qryqhzk

Reply:
Updated description The DAO token to build consensus among Hong Kong People